files/image_config/dhcp_relay/dhcp_relay.yml

```diff
@@ -0,0 +1 @@
+dhcp_servers: []
```

Comment:
Just to double confirm - have you verified that dhcp relay docker can start correctly with this empty value?